sdk/docs/pages/dapp-kit/sui-client-provider.mdx

```diff
@@ -140,3 +140,52 @@ function NetworkSelector() {
 	);
 }
 ```
+
+## Using network specific configuration
+
+If your dApp runs on multiple networks, the IDs for packages, and potentially other configuration
+will change depending on which network is being used. You can use `createNetworkConfigs` to create
+per-network config which can be accessed from your components.
+
+the `createNetworkConfigs` function returns the provided config, along with a `useNetworkConfig`
+hook you can use to get the config for the currently active network.
+
+```tsx
+import { createNetworkConfigs, SuiClientProvider, useSuiClientContext } from '@mysten/dapp-kit';
+
+import { createNetworkConfigs, SuiClientProvider, WalletProvider } from '@mysten/dapp-kit';
+import { type SuiClientOptions } from '@mysten/sui.js/client';
+import { QueryClient, QueryClientProvider } from '@tanstack/react-query';
+
+// Config options for the networks you want to connect to
+const { networkConfigs, useNetworkConfig } = createNetworkConfigs({
+	localnet: {
+		url: getFullnodeUrl('localnet')
+		myMovePackageId: '0x123',
+	},
+	mainnet: {
+		url: getFullnodeUrl('mainnet')
+		myMovePackageId: '0x456',
+	},
+});
+const queryClient = new QueryClient();
+
+function App() {
+	return (
+		<QueryClientProvider client={queryClient}>
+			<SuiClientProvider networks={networkConfigs} defaultNetwork="localnet">
+				<WalletProvider>
+					<YourApp />
+				</WalletProvider>
+			</SuiClientProvider>
+		</QueryClientProvider>
+	);
+}
+
+
+function YourApp() {
+	const { myMovePackageId } = useSuiClientContext();
```

Comment:
I find these docs a bit confusing - above we say we can use `useNetworkConfig` to retrieve the network config information, but we're using `useSuiClientContext` here. It also isn't obvious to me from the docs that I need to define and export my `createNetworkConfig` result in a separate file if I'd like to use the hook around the codebase. In the example above, we just create it with the top-level App definition and then never use it